lib/dockerize_rails/constants.rb

```diff
@@ -35,21 +35,36 @@ module Constants
         aliases: %I[undockerize ud du u dd],
         help: 'Removes docker configurations'.freeze,
         params: {
-          :'--purge' => "also removes #{Constants::DOCKERIZE_RAILS_CONFIG_FILE_NAME}".freeze
+          :'--purge' => "also removes #{CONFIG_FILE_NAME}".freeze
         }
       },
       docker_info: {
-        aliases: %I[docker_info di],
+        aliases: %I[docker_info info],
         help: 'Shows Docker information'.freeze
       },
       docker_pull: {
-        aliases: %I[docker_pull pull dp],
-        help: 'Pulls Docker images'.freeze
+        aliases: %I[docker_pull pull],
+        help: 'Pulls base Docker images (ruby, mysql/postgres)'.freeze
       },
       docker_build: {
-        aliases: %I[docker_build build db],
+        aliases: %I[docker_build build],
         help: 'Builds Docker images'.freeze
       },
+      docker_start: {
+        aliases: %I[docker_start start],
+        help: 'Run/Starts Docker containers'.freeze
+      },
+      docker_stop: {
+        aliases: %I[docker_stop stop],
+        help: 'Stops Docker containers'.freeze,
+        params: {
+            :'--delete' => 'also deletes the containers'.freeze
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.

<!-- Comment by [SideCI](https://sideci.com) -->